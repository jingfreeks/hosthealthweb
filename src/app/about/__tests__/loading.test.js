import { render } from "@testing-library/react";
import AboutLoading from "../loading";

describe("About Loading Page", () => {
  it("renders About Loading Page to match snaptshot", () => {
    const { container } = render(<AboutLoading />);
    expect(container).toMatchSnapshot();
  });
});
