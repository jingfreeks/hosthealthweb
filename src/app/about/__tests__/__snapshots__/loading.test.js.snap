// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Loading Page renders About Loading Page to match snaptshot 1`] = `
<div>
  <h1>
    Loading...
  </h1>
</div>
`;
